pkg/kubelet/container/runtime_cache.go

```diff
@@ -94,3 +94,25 @@ func (r *runtimeCache) getPodsWithTimestamp() ([]*Pod, time.Time, error) {
 	pods, err := r.getter.GetPods(false)
 	return pods, timestamp, err
 }
+
+// TestRunTimeCache embeds runtimeCache with some additional methods for testing.
```

Comment:
nit: I'd prefer placing this in a separate file `runtime_cache_testing.go` maybe?
Ideally, this should be compiled only for testing.